config/configuration.yml.example

```diff
@@ -94,6 +94,12 @@
 #
 # === More configuration options
 #
+# Disable default module:
+#
+# By default user may choose which modules can be disabled
+# this can be done in lib/open_project/configuration.rb
+# disabled_modules:[list_of_modules_name]
```

Comment:
Thanks for the feedback @machisuji. But let's leave this out of this PR, I think.